templates/category.twig

```diff
@@ -0,0 +1,43 @@
+{% extends "base.twig" %}
+
+{% block content %}
+
+	<div class="clearfix"></div>
+	<div class="skewed-overlay"></div>
+
+	<header class="page-header">
+		<div class="container">
+			<h1 class="page-header-title">{{ page_type.name }}</h1>
+			<div class="page-header-content">
+				<p>{{ page_type.description }}</p>
+			</div>
+		</div>
+	</header>
+
+	<div class="page-template">
+		<h3>{{ __( 'Results', 'planet4-master-theme' ) }}</h3>
+
+		<div class="row">
+			<div class="col-lg-8 multiple-search-result">
+				<ul class="list-unstyled">
+					{% for post in posts %}
+						{% include 'tease-page-type.twig' %}
+					{% endfor %}
+				</ul>
+			</div>
+		</div>
+
+		{% if posts.pagination.total > 1 %}
+			<div class="row">
+				<div class="col-lg-8">
+					<button
+						data-content=".multiple-search-result .list-unstyled"
+						data-page="1"
+						data-total="{{ posts.pagination.total }}"
+						data-url="{{ fn('get_term_link', page_type.term_id, 'p4-page-type') }}"
```

Comment:
also here please change this to generate category link
`data-url="{{ fn('get_category_link', category.term_id ) }}"`